control/region_requirements.proto

```diff
@@ -70,7 +70,7 @@ message SetSpectralRequirements {
     repeated SpectralConfig spectral_profiles = 3;
 
     message SpectralConfig {
-        // The required spectral coordinate (“z”), optionally preceded by a Stokes parameter (“I”, “Q”, “U” or “V”) If no Stokes parameter is present, or if the coordinate is empty, the active Stokes parameter is used.
+        // The required spectral coordinate (“z”), optionally preceded by a Stokes parameter. If no Stokes parameter is present, or if the coordinate is empty, the active Stokes parameter is used.
```

Comment:
I would change "Stokes parameter" to "polarization" here in the first two instances. Whether we should change it in the last instance depends on what the default is -- will it always be a Stokes parameter, or can the active polarization be something other than Stokes in this case?

Reply:
The active polarization may not be a Stokes parameter. I think we can change all of them to "polarization parameter"